docs/builders/clone.mdx

```diff
@@ -365,18 +368,30 @@ or responding to pattern `/dev/.+`. Example:
 
 - `machine` - (string) - Set the machine type. Supported values are 'pc' or 'q35'.
 
-#### VirtIO RNG device
```

Comment:
De-dented header size. The config reference section has h2 headers, so I think this must be h3.